static/src/javascripts/projects/commercial/modules/messenger/post-message.ts

```diff
@@ -1,7 +1,9 @@
 export const postMessage = (
 	message: Record<string, unknown>,
-	targetWindow: WindowProxy,
+	target: MessageEventSource,
 	targetOrigin?: string | null | undefined,
 ): void => {
-	targetWindow.postMessage(JSON.stringify(message), targetOrigin ?? '*');
+	target.postMessage(JSON.stringify(message), {
+		targetOrigin: targetOrigin ?? '*',
+	});
```

Comment:
Looks to me like `targetOrigin` should have a default value of `'*'`.

```suggestion
	target.postMessage(JSON.stringify(message), {
		targetOrigin: targetOrigin,
	});
```